### DISCUSSION TITLE
讓YouTube在iPad背景播放的方法 (免YT Premium)

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS3073-C/Python-Community-GS3073-C#601] [General]
Apple在iPad OS 16推出了一個新功能「螢前調度」(Stage Manager)，有Reddit上的網友發現可以透過這個新功能來實現YouTube 的背景播放，可以參考下方連結內影片的做法:

https://www.reddit.com/r/iPadOS/comments/wtk6lt/stage_manager_on_ipados_160_lets_you_listen_to/?utm_source=share&utm_medium=web2x&context=3

雖然可以透過這個小技巧實現背景播放，但像是無廣告、離線播放的功能還是要YouTube Premium才有，希望Google不會去修這個Bug XD。

**Comment by awehao** [comment]
該花的還是要花拉

**Comment by bww0401** [comment]
果真人類還是很聰明的(誤，雖然小弟沒有平板，但我自己也很喜歡邊看書邊聽youtube的音樂，所以我都是用電腦來同時做到這兩件事情，免費仔真香。

**Comment by manipulateed** [comment]
光背景音樂就夠讓人瞠目了，但其實在筆電來說，只要讓瀏覽器停在其他頁面，此時撥放的YT通常很少很少才有一個廣告。

**Comment by larry40305** [comment]
現在youtube的廣告越來越煩人，內容審查也越來越嚴格。但又不想買youtube preminum給官方錢，所以大家都在想辦法找其他方法。就看是youtube先改進，還是會先出現其他影音平台把它幹掉。

**Comment by 0212Jerry** [comment]
之前第一次用前台調度功能的時候非常不習慣，因為在做筆記的時候感覺畫面就會變小。而偏偏我的手指也不是特別靈巧，常常一不小心就滑到其他東西，於是我就把它關掉了。原po 的介紹讓我有新的認識。

**Comment by liangyinglly** [comment]
希望後台調度這個功能不要被Youtube破解，他好好用喔！

**Comment by Vincent20030314** [comment]
是個很實用的功能，窮人的救星，但感覺TY早晚會把這個bug處理掉。
